Gruntfile.js

```diff
@@ -82,6 +82,24 @@ module.exports = function (grunt) {
           '--copy-files'
         ],
       },
+      'copy-external': {
+        cmd: 'node',
+        args: [
+          'node_modules/@babel/cli/bin/babel',
+          'src/external/',
+          '--out-dir=out/src/external/',
+          '--copy-files'
+        ],
+      },
+      'copy-external-wpt': {
+        cmd: 'node',
+        args: [
+          'node_modules/@babel/cli/bin/babel',
+          'src/external/',
+          '--out-dir=out-wpt/src/external/',
+          '--copy-files'
+        ],
+      },
```

Comment:
I thought that Babel would pick up the `src/external/` JS files and "transpile" them into `out*/src/external/` as appropriate. If that's not the case, though, this is fine.

I just noticed I missed a `,.ts` in the `build-out-wpt` step. Maybe that would be all that's needed?

Reply:
Yup, that is exactly what the issue was, updated the patch.